cql/DashboardLibrary.cql

```diff
@@ -786,13 +788,47 @@ define function CollateConclusionCodes(D DiagnosticReport, ObsList List<Observat
   else
     ValuesFromObservation(D, ObsList)
 
+define function ToFhirCodeableConcept(c System.Code):
```

Comment:
I think this function worth to put in FHIRHelpers. This is helpful if we need do similar convert in other modules.

Reply:
Good idea, I think it is generic enough to move there. I did this in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/117/commits/938d71d318b5a23471dc3c0d8a7c1b81b3eebe2e.